src/settings/languages.js

```diff
@@ -40,22 +40,30 @@ const languages: $ReadOnlyArray<Language> = [
   { locale: 'cs', name: 'Czech', nativeName: 'Čeština' },
   { locale: 'en', name: 'English', nativeName: 'English' },
   { locale: 'es', name: 'Spanish', nativeName: 'Español' },
+  { locale: 'fa', name: 'Persian', nativeName: 'فارسی' },
+  { locale: 'fi', name: 'Finnish', nativeName: 'Suomi' },
   { locale: 'fr', name: 'French', nativeName: 'Français' },
   { locale: 'de', name: 'German', nativeName: 'Deutsch' },
+  { locale: 'gl', name: 'Galician', nativeName: 'Galego' },
   { locale: 'hi', name: 'Hindi', nativeName: 'हिन्दी' },
   { locale: 'hu', name: 'Hungarian', nativeName: 'Magyar' },
   { locale: 'id-ID', name: 'Indonesian', nativeName: 'Bahasa Indonesia' },
   { locale: 'it', name: 'Italian', nativeName: 'Italiano' },
   { locale: 'ja', name: 'Japanese', nativeName: '日本語' },
+  { locale: 'ko', name: 'Korean', nativeName: '한국어' },
```

Comment:
https://github.com/zulip/zulip-mobile/pull/3897/commits/e5ac53299bda6eaa7e3bc94815e405794b36f8ff also adds Lithuanian and no others. (Your choice to leave out Lithuanian may have been intentional, as a consequence of using a particular threshold that Greg didn't use.)

Reply:
Exactly that. Serbian and Swedish already fall below that arbitrary threshold, though, so it makes sense to lower it. (Done.)